var/spack/repos/builtin/packages/cns-nospec/package.py

```diff
@@ -49,6 +50,17 @@ class CnsNospec(MakefilePackage):
     build_directory = 'MiniApps/CNS_NoSpec'
 
     def edit(self, spec, prefix):
+        def_file = FileFilter('Tools/F_mk/GMakedefs.mak')
+        def_file.filter('tdir = t/.*', 'tdir := t/$(suf)')
+        def_file.filter('hdir = t/.*', 'hdir := t/html')
+        def_file.filter('include $(BOXLIB_HOME)/Tools/F_mk/GMakeMPI.mak', '#')
+
+        if '+mpi' in spec:
+            def_file.filter('FC.*:=.*', 'FC = {0}'.format(spec['mpi'].mpifc))
+            def_file.filter('F90.*:=.*', 'F90 = {0}'.format(spec['mpi'].mpifc))
+            def_file.filter('mpi_include_dir =.*', 'mpi_include_dir = {0}'.format(join_path(spec['mpi'].prefix), 'include'))
+            def_file.filter('mpi_lib_dir =.*', 'mpi_lib_dir = {0}'.format(join_path(spec['mpi'].prefix), 'lib'))
```

Comment:
You can use `spec['mpi'].prefix.include` and `spec['mpi'].prefix.lib` here to make it a little shorter.